konfera/models/order.py

```diff
@@ -53,3 +53,10 @@ def to_pay(self):
     @property
     def variable_symbol(self):
         return str(int(self.uuid))[:10]
+
+    @property
+    def event(self):
+        if len(self.ticket_set.all()) > 0:
```

Comment:
This makes 2 queries to the DB. 

either "precache" the query by storing it in a variable (just 1 query to DB):
```
tickets = self.ticket_set.all()
if len(tickets):
    return tickets[0].type.event
```

or use 2 lightweight queries (in could be faster in this situation)
```
if self.ticket_set.exists():
    return self.ticket_set.first().type.event
```